.github/workflows/build.yml

```diff
@@ -37,21 +37,21 @@ jobs:
     steps:
       - uses: actions/checkout@v2
 
-      - name: Use Node 12.X
+      - name: Use Node 16.X
         uses: actions/setup-node@v2
         with:
-          node-version: 12.x
+          node-version: 16.x
           cache: yarn
 
       - run: yarn install --frozen-lockfile
 
-      - run: yarn test --coverage
+      - run: yarn workspace @itwin/itwinui-react test --coverage
```

Comment:
Maybe we should move commands like these to the root package.json. What I mean: `"test": "yarn workspace @itwin/itwinui-react test"`

Reply:
we already have `test` in root package.json for running `test` in all repos

Reply:
So why not use it?

Reply:
it doesn't accept `--coverage` flag